dotCMS/src/integration-test/java/com/dotcms/security/apps/AppsAPIImplTest.java

```diff
@@ -242,13 +242,13 @@ public void Test_Save_Secrets_Then_Update_Single_Secret_Verify_Updated_Value() t
 
     @Test
     public void Test_Save_Secrets_Then_Delete_Single_Secret_Entry_Then_Add_It_Again_With_New_Value_Then_Verify() throws DotDataException, DotSecurityException {
-        final ServiceIntegrationAPI api = APILocator.getServiceIntegrationAPI();
+        final AppsAPI api = APILocator.getAppsAPI();
         final User admin = TestUserUtils.getAdminUser();
         final Host host = new SiteDataGen().nextPersisted();
 
         //Let's create a set of secrets for a service
-        final ServiceSecrets.Builder builder1 = new ServiceSecrets.Builder();
-        final ServiceSecrets secrets1 = builder1.withServiceKey("serviceKeyHost-1")
+        final AppSecrets.Builder builder1 = new AppSecrets.Builder();
+        final AppSecrets secrets1 = builder1.withKey("appKeyHost-1")
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The String literal "appKeyHost-1" appears 7 times in this file; the first occurrence is on line 251](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5218098)